test-runtime/interval.test.ts

```diff
@@ -115,7 +115,7 @@ describe('interval selections at runtime in unit views', function() {
 
     embed(spec('unit', 1, {type, encodings: ['x']}, {values, x: {type: 'temporal', timeUnit: 'day'}}));
 
-    extents = [[1136188800000, 1136275200000, 1136361600000], [1136448000000, 1136534400000]];
+    extents = [[1136187936000, 1136361600000], [1136447136000, 1136535264000]];
```

Comment:
@arvind -- I have to make change to this line to make it pass the runtime test -- but not sure if it is really correct. Could you please help double check?

(I also don't understand why the old extent should have 3 members.)

Reply:
LGTM. The old extents had 3 members because I think ordinal-point scales were being generated by timeUnits? Thus, inverting brush extents over a range returned the constituent points.